controllers/commentController.js

```diff
@@ -0,0 +1,38 @@
+const {
+    addComment,
+    fetchComments,
+    removeComment
+} = require('../models/commentModel')
+
+exports.getComments = (req, res, next) => {
```

Comment:
You've moved this into its own controller file but left the original function in the review controller